source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,245 @@
+.. _realm-sync-migrate-modes:
+
+=========================
+Migrate Device Sync Modes
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 3
+   :class: singlecol
+
+Atlas App Services Apps can have one of two Device Sync modes: Partition-Based Sync
+and Flexible Sync. Partition-Based Sync is an older mode and you should consider
+migrating to Flexible Sync.
+
+Migrating an App that uses Partition-Based Sync to Flexible Sync is an automatic
+process. Other than upgrading the SDK version, migrating sync modes doesn't require 
+any changes to your client app code. Your App's backend will automatically translate
+the client's Partition-Based Sync requests into Flexible Sync.
+
+Migrating results in Partition-Based Sync clients communicating with a
+Flexible Sync Atlas App Services App.
+
+Requirements
+------------
+
+Your App Services App:
+
+- Must currently use Partition-Based Sync
+
+Linked Atlas Cluster:
+
+- Must be running mongodb 5.0+ for Flexible Sync compatibility
+- Must have storage auto-scaling enabled
+- Must be M10+ to access this setting
+- Must have time-based oplog setting enabled
+
+Client Apps:
+
+- ``realm-core`` vTBD+, which means these SDK versions: TBD
```

Comment:
@kmorkos, do we know what version of realm-core and SDK versions users will need?

Reply:
Not yet, tagging @michael-wb since he's currently working on the realm-core side